### DISCUSSION TITLE
Updated Version Secure Realtime Chat App (Signal Clone)

### DISCUSSION THREAD

**Discussion by Supsource** [notJust-dev/notjustdev#218] [Ideas]
yeah! I know, it's already there in the channel. But the thing is that the video (series to be honest) is more than 2 years old, and there is no doubt that [video series](https://www.youtube.com/playlist?list=PLY3ncAV1dSVDBVPIKBlaRJGAniKRA37Zc) is even now one of the BEST(when I say BEST, I mean BEST✨) video on YouTube about building a Secure Realtime Chat App in React Native.

In those years a lot of changes(in the expo, react native, and AWS) have been made, and from a beginner's point of view it's really hard to follow along with, so if we have an Updated version of that series, that will help lots of upcoming developers along with me. 

To make things more interesting, here few feature suggestions : 

1. All the features from Session 1
2. Secure authentication
3. WhatsApp status or Instagram story kind of thing. [for reference.](https://github.com/iguilhermeluis/react-native-stories-media)
4. In-app calling system. [A github repo might help](https://github.com/CleverTap/clevertap-signed-call-react-native-sdk) and [using Stream](https://getstream.io/blog/react-native-video-calling-app/)

One last message from my side, 
Thank you a lot, Vadim. It’s kind of weird and sad that you don’t even properly know the amount of contribution you have made to a developer like me. I’m a self-taught developer who just graduated from high school and is doing pretty well in freelancing. The person I am today as a developer, without you, I don't think it would be possible. Again, thank you so much, keep up the great work.

**Comment by VadimNotJustDev** [comment]
That's a great idea, thank you for sharing and adding also details. 

The last part of your message made my day <3

**Comment by joeymag** [comment]
maybe use supabase

**Comment by OvitalyO** [comment]
We're really looking forward to it

**Comment by DID0101** [comment]
i cantconfigure errors can yall did it ?
